src/components/results/dynamicsimulation/dynamic-simulation-result-chart.js

```diff
@@ -289,11 +300,6 @@ const DynamicSimulationResultChart = ({
 
             // set the current plot id in scaling
             setPlotIdScale(plotScale ? plotId : undefined);
-
-            // auto switch in full view
-            if (plotScale) {
-                setFullView(plotScale);
-            }
```

Comment:
Is this related to the translations ?

Reply:
No, this is a tiny bonus which is noted the the bottom of description..
Why i have to do this in the PR? for making be more coherent to the label 'Enlarge the graph'